Controller/Adminhtml/Import/Ajax.php

```diff
@@ -101,15 +100,14 @@ public function execute()
                 case 'orders':
                     $serializedOrders  = $this->serializeRecords($this->orderData->getOrders($storeId, $chunkId), $this->orderSerializer);
                     $result['success'] = $client->orderBatch($serializedOrders);
+                    if ($chunkId == (int)$this->request->getParam('ordersChunks') - 1 && $importType == 'orders') {
```

Comment:
Same Q

Reply:
In this case importType is used to determine when to mark the end of the importing process. This can be achieved by checking for the importType witch should be orders - meaning last synchronisation step has started.